.github/workflows/performance-release-report.yml

```
@@ -22,14 +22,12 @@ on:
       rc_label:
         description: 'RC Label'
         required: false
-  repository_dispatch:
-     types: [perf-report]
 

```

```suggestion

permissions:
  contents: read
  
```
Keeping permissions minimal is good practice.